.github/CODEOWNERS

```
@@ -14,6 +14,7 @@ src/cli/commands/test @snyk/snyk-open-source
 src/cli/commands/monitor @snyk/snyk-open-source
 src/cli/commands/test/formatters/format-reachability.ts  @snyk/tardis
 help/  @snyk/hammer
+help/cli-commands/iac*.md @snyk/group-infrastructure-as-code
```

Would you do shared ownership over this?

```suggestion
help/cli-commands/iac*.md @snyk/group-infrastructure-as-code @snyk/hammer 
```

Those pages are coming over from Gitbook User docs, so when we touch those files here, it's usually something small/chore https://github.com/snyk/snyk/tree/master/help#cli-help-files

---

I didn't know that was an option, that sounds totally fine for me. Do you reckon it's sensible that we're going "backwards" in the new workflow, and will sync to gitbook later? This is a entitlement-gated, and currently that can't be used by any customer, so we didn't want to publish gitbook pages up-front.

---

Yup, we designed it with this exact flow in mind. As long as the file is not in this folder: https://github.com/snyk/user-docs/tree/main/docs/features/snyk-cli/commands it won't be affected or synced in Gitbook